client/src/components/TodoList.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import '../App.css';
+import graphql from 'babel-plugin-relay/macro';
+import TodoListItem from './TodoListItem';
+import { TodoList_query$key } from './__generated__/TodoList_query.graphql';
+import { useFragment } from 'react-relay/hooks';
+// Type ctrl+space on the line that uses the thing you want to import and it will pop up here
+
+
+
+type Props = {
+    
+    query: any
```

Comment:
Relay doesn't like it if you spread a root query so this is what Dylan and I came up with